spec/src/main/asciidoc/core/injectionandresolution_full.asciidoc

```diff
@@ -88,14 +88,17 @@ The container automatically detects inconsistent specialization and treats it as
 
 ==== Inter-module injection in {cdi_full}
 
-In addition to rules defined in <<inter_module_injection>>, the following rules apply.
+{cdi_full} extends the set of rules defined in <<inter_module_injection>>.
+Below is a complete set of rules.
```

Comment:
```suggestion
Instead of the rules in <<inter_module_injection>>, the following rules apply in {cdi_full}.
```

Reply:
Hmm, I had it worded similarly originally. However, that makes it sounds like the set of rules is different when it is really just a superset. I can change it to your suggestion though, I don't mind either.

Reply:
Or not :-) My suggestion also works :-)